extensions/amp-call-tracking/0.1/amp-call-tracking.js

```diff
@@ -82,15 +84,17 @@ export class AmpCallTracking extends AMP.BaseElement {
       .expandUrlAsync(user().assertString(this.configUrl_))
       .then(url => fetch_(this.win, url))
       .then(data => {
-        userAssert(
-          'phoneNumber' in data,
-          'Response must contain a non-empty phoneNumber field %s',
-          this.element
-        );
-
-        this.hyperlink_.setAttribute('href', `tel:${data['phoneNumber']}`);
-        this.hyperlink_.textContent =
-          data['formattedPhoneNumber'] || data['phoneNumber'];
+        if (data['phoneNumber']) {
+          this.hyperlink_.setAttribute('href', `tel:${data['phoneNumber']}`);
+          this.hyperlink_.textContent =
+            data['formattedPhoneNumber'] || data['phoneNumber'];
+        } else {
+          user().warn(
+            TAG,
+            'Response must contain a non-empty phoneNumber field %s',
```

Comment:
The message still looks like an error. How about:

> Response does not have phoneNumber field. Call tracking not applied.

Reply:
Sounds good! Will do this, and merge 😄